tensorflow/docs_src/community/documentation.md

```diff
@@ -148,18 +148,30 @@ viewing. Do not include url parameters in the source code URL.
 Before building the documentation, you must first set up your environment by
 doing the following:
 
-1. If bazel is not installed on your machine, install it now. If you are on
+1. If pip isn't installed on your machine, install it now by issuing the
+following command:
+
+        $ sudo easy_install pip
+
+2. Use pip to install mock and pandas by issuing the following
+   command (Note: If you are using
+   a [virtualenv](https://virtualenv.pypa.io/en/stable/) to manage your
+   dependencies, you may not want to use sudo for these installations):
+
+        $ sudo pip install mock pandas
+
+3. If bazel is not installed on your machine, install it now. If you are on
```

Comment:
But I've looked at this section a little harder and, while it used to be necessary, it is not any longer.

I've test built the docs from a clean `virtualenv` and gotten zero `diff`.

* `mock` is only necessary for testing
* The `pandas`-dependant functions are now defined whether or not `pandas` is installed.
  So since the docs don't need to run those functions there's no problem.

So, I think it's fine to submit this without this latest commit.